Game/CMakeLists.txt

```diff
@@ -51,4 +51,5 @@ target_link_libraries(<% gameName %> ${GAME_DEPS})
 configure_file(Config.txt ${AssetOutputDir}/Config.txt)
 configure_file(save.dat ${AssetOutputDir}/save.dat COPYONLY)
 file(COPY maps/ DESTINATION ${AssetOutputDir}/maps/)
-file(COPY project/ DESTINATION ${AssetOutputDir}/project/)
```

Comment:
Instead of copying all of the project folder you should explicitly copy collision-types.json.